

message RobotPositionMessage{
	repeated SensorData sensorData   	= 1;
	required uint32 TimeDiff = 2;
}

message SensorData {
	required float  SensorValue = 1;
	required float SensorValueDiff = 2;
}

message AllSensorValuesMessage{
	repeated SensorData jointData   	= 1;
	repeated SensorData sensorData		= 2;
	required uint32 TimeDiff = 3; 
}
